packages/features/calendars/DatePicker.tsx

```diff
@@ -190,10 +205,7 @@ const Days = ({
               onClick={() => {
                 props.onChange(day);
               }}
-              disabled={
-                (includedDates && !includedDates.includes(yyyymmdd(day))) ||
```

Comment:
includedDates can actually be null now(while slots request is loading). Earlier, it was an empty array always.